dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServerTest.java

```diff
@@ -235,12 +233,16 @@ public void testServletContextMajorMinorVersion() throws Exception {
         String majorVersionExpectedResult = "majorVersion: 4";
         if (JakartaEEAction.isEE9Active()) {
             majorVersionExpectedResult = "majorVersion: 5";
-        } else if (JakartaEEAction.isEE10OrLaterActive()) {
+        } else if (JakartaEEAction.isEE10Active() || JakartaEEAction.isEE11Active()) {
```

Comment:
Use isEE10OrLaterActive instead?

Reply:
Yea, not sure why I changed that...

Reply:
Ah, ok I remember, I did this because I think it is unlikely that we'll have a Servlet 6.2, I guess it is possible. I figured I'd just code it this way for now since we know EE10 and EE11 are going to have a major version of 6.

Reply:
I don't mind reverting this back for now.